godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -8,13 +8,45 @@
 
 import Foundation
 import UIKit
+import SWXMLHash
 
 class TractViewController: BaseViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
         displayTitle()
+        initializeView()
+        setupSwipeGestures()
+    }
+    
+    func setupSwipeGestures() {
+        let swipeLeft = UISwipeGestureRecognizer(target: self, action: #selector(handleGesture))
+        swipeLeft.direction = .left
+        self.view.addGestureRecognizer(swipeLeft)
+        
+        let swipeRight = UISwipeGestureRecognizer(target: self, action: #selector(handleGesture))
+        swipeRight.direction = .right
+        self.view.addGestureRecognizer(swipeRight)
+    }
+    
+    @objc fileprivate func handleGesture(gesture: UISwipeGestureRecognizer) {
+        if gesture.direction == .right {
+            moveToPreviousPage()
+        } else if gesture.direction == .left {
+            moveToNextPage()
+        }
+    }
+    
+    func moveToNextPage() {
```

Comment:
Are we going to push/pop view controllers? I don't think this will work because we will need to be able to jump between views based on button presses later on. I was thinking we would do one view controller with a collection of views, or something like that.